[max] Sending action rebootCube causes StringIndexOutOfBoundsException

Rebooting the cube from either paperUI, Habpanel or via REST API will work, but cause a StringIndexOutOfBoundsException. 

## Expected Behavior
Exception should not happen.

## Current Behavior
Cube will be rebooted, but an exception is logged:
```
[ERROR] [st.core.internal.thing.ThingResource] - Exception during HTTP PUT request for update config at 'things/max:bridge:NEQ1442746/config'
java.lang.StringIndexOutOfBoundsException: String index out of range: -3
	at java.lang.String.substring(String.java:1931) [?:?]
	at org.openhab.binding.max.internal.command.UdpCubeCommand.receiveUdpCommandResponse(UdpCubeCommand.java:139) [205:org.openhab.binding.max:2.3.0]
[...]
```

## Possible Solution
The exception is caused by a wrong substring offset in org.openhab.binding.max.internal.command.UdpCubeCommand

The offset for should be 20, not 24. Since the string is only 21 chars long, an exception is thrown.

## Steps to Reproduce (for Bugs)
Reboot cube via PaperUI, Habpanel, or REST API

## Context
Not a big deal really.

## Your Environment
openHABian, openHAB 2.4

--- a/addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/command/UdpCubeCommand.java
+++ b/addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/command/UdpCubeCommand.java
@@ -112,8 +112,10 @@ public class UdpCubeCommand {
                 // We have a response
                 String message = new String(receivePacket.getData(), receivePacket.getOffset(),
                         receivePacket.getLength(), StandardCharsets.UTF_8);
-                logger.trace("Broadcast response from {} : {} '{}'", receivePacket.getAddress(), message.length(),
-                        message);
+                if (logger.isDebugEnabled()) {
+                    logger.debug("Broadcast response from {} : {} '{}'", receivePacket.getAddress(), message.length(),
+                            message);
+                }
 
                 // Check if the message is correct
                 if (message.startsWith("eQ3Max") && !message.equals(MAXCUBE_COMMAND_STRING)) {
@@ -133,7 +135,7 @@ public class UdpCubeCommand {
                     } else {
                         // TODO: Further parsing of the other message types
                         commandResponse.put("messageResponse",
-                                Utils.getHex(message.substring(24).getBytes(StandardCharsets.UTF_8)));
+                                Utils.getHex(message.substring(20).getBytes(StandardCharsets.UTF_8)));
                     }
 
                     commandRunning = false;
@@ -159,7 +161,7 @@ public class UdpCubeCommand {
      * Send broadcast message over all active interfaces
      *
      * @param commandString string to be used for the discovery
-     * @param ipAddress IP address of the MAX! Cube
+     * @param ipAddress     IP address of the MAX! Cube
      *
      */
     private void sendUdpCommand(String commandString, String ipAddress) {

